internal/pkg/client/progress.go

```diff
@@ -97,7 +97,7 @@ func CopyWithContext(ctx context.Context, dst io.Writer, src io.Reader) error {
 	return err
 }
 
-// DownloadProgressBar is used for chunked library downloads.
+// DownloadProgressBar is used for chunked scs-library-client downloads.
```

Comment:
@DrDaveD reference should be to the new community library

Reply:
Fixed